liquibase-core/src/main/java/liquibase/command/AbstractCliWrapperCommandStep.java

```diff
@@ -35,6 +36,19 @@ public final void run(CommandResultsBuilder resultsBuilder) throws Exception {
         }
         resultsBuilder.addResult("statusCode", statusCode);
 
+/*
```

Comment:
This is just turns-out-unneeded code that should be removed before merging, right?

Reply:
Removed